juniper_official/SRX/Security/srx-cluster-flow-session-summary.rule

```diff
@@ -0,0 +1,225 @@
+/*
+ * Detects flow session details for each PIC and notifies when anomalies are found.
+ * 
+ * 
+ *   1) "active-sessions-threshold-value" is the threshold that causes the rule to report
+ *      an anomaly.  By default it's 5000000. This rule will set a dashboard
+ *      color to red when active sessions for a PIC slot is greater than this value.
+ *   2) "max-sessions-threshold-value" is the threshold that causes the rule to report
+ *      an anomaly.  By default it's 5000000. This rule will set a dashboard
+ *      color to red when active sessions for a PIC slot is greater than this value.
+ *  
+ */
+iceberg {
+    topic security  {
+        rule check-flow-session-summery {
+            description "Monitors and notify flow session details";
+            synopsis "flow session summery statistics analyzer"; 
```

Comment:
- Start descriptions and synopsis with CAPS letter in all rules
- s/summery/summary/g